vendor/engines/split_accounts/app/models/split_accounts/account_extension.rb

```diff
@@ -1,11 +1,13 @@
 module SplitAccounts
-  module AccountExtensions
+  module AccountExtension
 
     extend ActiveSupport::Concern
 
     included do
       has_many :parent_splits, class_name: "SplitAccounts::Split", foreign_key: :subaccount_id, inverse_of: :subaccount
       has_many :parent_split_accounts, through: :parent_splits
+
+      scope :excluding_split_accounts, -> { where("type != ?", "SplitAccounts::Split") }
```

Comment:
Normally with rails 4 I would use `where.not(...)`.  For rails 3.2 will this work, or should I use Arel to better support Oracle?